HipChatPS/Public/Send-HipChat.ps1

```diff
@@ -0,0 +1,92 @@
+﻿function Send-HipChat {
```

Comment:
`-HipChat` is the prefix of the module.
this function should be renamed to `Send-HipChatMessage` (or something to that effect)